jsBasics/task4-3.js

```diff
@@ -0,0 +1,8 @@
+let radius = (Math.random()*20).toFixed(2)
+console.log ('Random radius of cylinder is ', radius)
+let height = (Math.random()*20).toFixed(2)
+console.log ('Random height of cylinder is ', height)
+const pi = Math.PI.toFixed(3)
+const squareRadius = Math.pow(radius, 2)
```

Comment:
it is not suitable name for this value. It is not a radius but radius to the power of 2

Reply:
I would not create dedicated variable for this but if you want you can name it like `radiusPoweredTo2`